packages/boot/tools/supports.ts

```diff
@@ -477,7 +477,7 @@ export const makeSwingsetTestKit = async (
       case `${BridgeId.VTRANSFER}:IBC_METHOD`: {
         switch (obj.method) {
           case 'startChannelOpenInit':
-            pushInbound(BridgeId.DIBC, icaMocks.channelOpenAck(obj));
```

Comment:
>- some contracts rely on ICAs to be created before a `StartFn` completes

How can that be guaranteed in production?

Reply:
I'm not sure it can be. Is there a time bound for how long a `startFn` can take?

I at least verified the startFn completes in multichain-testing, but that has shorter block times and maybe has less latency .

Reply:
On this commit - another approach here is to make a parameter for `makeWalletFactoryContext` / `makeSwingsetTestKit`. Maybe something like:

```js
export type BridgeTestOpts = Record<
  BridgeId,
  { ackChannelOpenImmediately?: boolean }
>;
```

There are some tests I missed that are failing. I think most can be amended to send and upgrade during `sendPacket` events (transfer, delegate).

Thoughts on which is preferable?